web/src/pages/your_account/mod.rs

```diff
@@ -164,9 +164,25 @@ impl Component for YourAccount {
                 of the network. Click on any name, send some asset and see your
                 transaction being validated.
             " } <br /> { "
-                The most recent transactions will appear below, if you see it
-                changing rapidly, that's probably because someone else is
-                running a speedtest.
+                The most recent transactions will appear below, with the most
+                recent on top.
+                If you see it changing rapidly, that's probably because
+                someone else is running a speedtest.
+            " } </p>
+
+            <p> { "
+                One of the specificity of AT2 is that there isn't a regular
+                synchronisation of the network.
+                So when one is sending a transaction that debits more than one
+                have, we can't really say it's invalid. There might be
```

Comment:
I don't understand this. Why do you talk about this here? When you send a transaction, you usually know how much assets are in your account, no?

Reply:
> When you send a transaction, you usually know how much assets are in your account, no?

Yes and no, as pretty much everything is async, until the network settled, it's not evident to know if a tx is debitting too much because it arrived too early or if one tries to draw too much.

But yeah, that's a bit confusing: dropping it and capping the input to the balance of the user.